src/os_abstract.cpp

```diff
@@ -393,7 +393,7 @@ int os_set_affinity(const os_thread_t &thread, const os_cpuset_t &_mycpuset) {
     return 0;
 }
 
-int os_get_max_active_fds_num() {
+int os_get_max_fds_num() {
```

Comment:
I found a small bug in error handler below:
```return 1024; // try the common default```
must be
```return max_active_fd_num = 1024; // try the common default```
otherwise we'll call `getrlimit` every time.